Lib/test/test_cgi.py

```diff
@@ -363,6 +363,14 @@ def test_fieldstorage_as_context_manager(self):
         with self.assertRaisesRegex(ValueError, 'I/O operation on closed file'):
             fs.file.read()
 
+    def test_fieldstorage_exit_context(self):
+        fs = cgi.FieldStorage()
+        try:
```

Comment:
I don't think there's a need to use `try` / `except` here in the unit test.

Reply:
should I remove it? I was assuming it should assert the test.